src/python_bindings/examples/mine_typos.py

```diff
@@ -1,11 +1,10 @@
 """Typo mining example using Desbordante algorithms."""
 
-from itertools import groupby, islice
-
-from colorama import Style, Fore
-from jellyfish import levenshtein_distance
 import desbordante as desb
 import pandas as pd
+from colorama import Style, Fore
+from itertools import groupby, islice
+from jellyfish import levenshtein_distance
```

Comment:
Why? The import order was correct, see [PEP-8](https://peps.python.org/pep-0008/#imports)

Reply:
Oh, CLion uses a different format, reverted.